app/views/providers/show.html.erb

```diff
@@ -20,12 +20,28 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <% if @provider.rolled_over? %>
-      <h2 class="govuk-heading-m"><%= link_to "Current cycle (2019 – 2020)", provider_recruitment_cycle_path(@provider.provider_code, '2019'), data: { qa: 'provider__courses__current_cycle' }, class: "govuk-link" %></h2>
+      <h2 class="govuk-heading-m">
+        <%= govuk_link_to @current_recruitment_cycle.title,
```

Comment:
No need to use `govuk_link_to` here, it just adds another level of abstraction IMO